### PR TITLE
Updating Change logs to link to SDK repos

### PR DESCRIPTION
**Pull request by mrm9084**
We were just duplicating the change logs, redirecting to SDK repo after talking with Zhenlan.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,9 @@ Module | Platform | Sample | Release Notes
...
+[azure-appconfiguration-provider](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/appconfiguration/azure-appconfiguration-provider)<br/>[![Pypi](https://img.shields.io/pypi/v/azure-appconfiguration-provider.svg?color=blue)](https://pypi.org/project/azure-appconfiguration-provider/) | Python | [Sample](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/appconfiguration/azure-appconfiguration-provider/samples) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/AzureAppConfigurationProviderPython.md))
```
Comment (zhenlan) [inline README.md:37]: There is a redundant `)` at the end of the line.

File: README.md [hunk 2]
```diff
@@ -32,9 +32,9 @@ Module | Platform | Sample | Release Notes
...
+[azure-spring-cloud-appconfiguration-config](https://github.com/Azure/azure-sdk-for-java/tree/main/sdk/appconfiguration/azure-spring-cloud-appconfiguration-config)<br/>[![Maven Central](https://img.shields.io/maven-central/v/com.azure.spring/azure-spring-cloud-appconfiguration-config.svg?color=blue)](https://search.maven.org/artifact/com.azure.spring/azure-spring-cloud-appconfiguration-config) | Spring Boot | [Sample](https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/appconfiguration/azure-appconfiguration-sample) | [Release Notes](https://github.com/Azure/AppConfiguration/blob/main/releaseNotes/SpringCloudAzureAppConfigurationConfig.md)
```
Comment (zhenlan) [inline README.md:35]: Should the module name be updated too?

File: releaseNotes/SpringCloudAzureAppConfigurationConfig.md [hunk 3]
```diff
@@ -1,12 +1,34 @@
...
+[Source code][source_code] | [Package (Maven)][package]
+
+[CHANGELOG](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/spring-cloud-azure-appconfiguration-config/CHANGELOG.md)
 
 # spring-cloud-azure-appconfiguration-config-web
 
-[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+[Source code][source_code_web] | [Package (Maven)][package_web]
+
+## 4.0.0 - March 29, 2023
```
Comment (zhenlan) [inline releaseNotes/SpringCloudAzureAppConfigurationConfig.md:17]: I thought it was 4.7.0. Do I miss something?

File: releaseNotes/AzureSpringCloudFeatureManagement.md [hunk 4]
```diff
@@ -27,15 +27,15 @@
...
 * TimeWindowFilter now supports ISO 8601 Dates.
 
 <!-- LINKS -->
-[docs]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-spring-cloud-feature-management
+[docs]: https://github.com/Azure/azure-sdk-for-java/tree/azure-spring-cloud-feature-management-web_2.10.0/sdk/appconfiguration/azure-spring-cloud-feature-management
 [package]: https://mvnrepository.com/artifact/com.azure.spring/azure-spring-cloud-feature-management
 [samples]: https://github.com/Azure-Samples/azure-spring-boot-samples/tree/main/appconfiguration/feature-management-sample
-[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/appconfiguration/azure-spring-cloud-feature-management
+[source_code]: https://github.com/Azure/azure-sdk-for-java/tree/azure-spring-cloud-feature-management-web_2.10.0/sdk/appconfiguration/azure-spring-cloud-feature-management
```
Comment (zhenlan) [inline releaseNotes/AzureSpringCloudFeatureManagement.md:37]: So it's correct that this is 2.10 and the other is 2.11?

File: releaseNotes/SpringCloudAzureAppConfigurationConfig.md [hunk 5]
```diff
@@ -1,12 +1,34 @@
...
+* Feature Flags can now be selected using a key and label filter, instead of just a label filter. The configurations are now `spring.cloud.azure.appconfiguration.stores[0].feature-flags.selects[0].key-filter` and `spring.cloud.azure.appconfiguration.stores[0].feature-flags.selects[0].label-filter`.
+
+### Breaking Changes
+
+* Libraries and namespaces have been renamed to `spring-cloud-azure-appconfiguration-config` and `com.azure.spring.cloud.appconfiguration.config`.
+* Renamed `ConfigurationClientBuilderSetup` to `ConfigurationClientCustomizer`.
+* Renamed `SecretClientBuilderSetup` to `SecretClientCustomizer`.
+* Removed `AppConfigurationCredentialProvider` and `KeyVaultCredentialProvider`, instead you can use [Azure Spring common configuration properties][azure_spring_common_configuration_properties] or modify the credentials using `ConfigurationClientCustomizer`/`SecretClientCustomizer`.
+* Feature Flags are now merged when found across multiple stores, if duplicate keys are found the highest priority store wins.
```
Comment (zhenlan) [inline releaseNotes/SpringCloudAzureAppConfigurationConfig.md:31]: ... when loaded from multiple stores, ...

File: releaseNotes/SpringCloudAzureFeatureManagement.md [hunk 6]
```diff
@@ -2,11 +2,15 @@
...
 [Source code][source_code] | [Package (Maven)][package] | [Samples][samples] | [Product documentation][docs]
 
+[CHANGELOG](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/spring-cloud-azure-feature-management/CHANGELOG.md)
+
 # spring-cloud-azure-feature-management-web
 
-[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
+[Source code][source_code_web] | [Package (Maven)][package_web]
+
+### Breaking Changes
```
Comment (zhenlan) [inline releaseNotes/SpringCloudAzureFeatureManagement.md:11]: What's the version?

File: releaseNotes/AzureSpringCloudAppConfigurationConfig.md
Comment (zhenlan) [inline releaseNotes/AzureSpringCloudAppConfigurationConfig.md:1]: We probably should add a note at the beginning indicating that the active development has moved to another module/version and provide links. Same for the feature management one.

File: releaseNotes/AzureSpringCloudFeatureManagement.md (hunk 4 above)
Comment (mrm9084) [reply releaseNotes/AzureSpringCloudFeatureManagement.md:37]: Yes, the feature management library didn't have a release every time the provider library did.

File: releaseNotes/AzureSpringCloudFeatureManagement.md [hunk 7]
```diff
@@ -4,7 +4,9 @@
 
 # azure-spring-cloud-feature-management-web
 
-[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
+[Source code web][source_code_web] | [Package (Maven) web][package_web] | [Samples web][samples_web] | [Product documentation][docs]
+
+> Active development has moved to group `com.azure.spring` and artifacts `spring-cloud-azure-feature-management` and  `spring-cloud-azure-feature-management-web`. See [SpringCloudAzureFeatureManagement](.\SpringCloudAzureFeatureManagement.md) for more information.
```
Comment (zhenlan) [inline releaseNotes/AzureSpringCloudFeatureManagement.md:9]: Should we add a similar note to AzureSpringCloudAppConfigurationConfig.md?

File: releaseNotes/SpringCloudAzureAppConfigurationConfig.md [hunk 8]
```diff
@@ -1,12 +1,34 @@
...
-[Source code web ][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+[Source code][source_code_web] | [Package (Maven)][package_web]
+
+## 4.7.0 - April 06, 2023
+
+### Features Added
 
-[Latest Library](./AzureSpringCloudAppConfigurationConfig)
+* Added support for [Azure Spring common configuration properties][azure_spring_common_configuration_properties].
+* Added support for Geo-Replication, enables the library to connect to a replica when the origin store fails to connect. Configuration is done through the `spring.cloud.azure.appconfiguration.stores[0].endpoints` property or `spring.cloud.azure.appconfiguration.stores[0].connection-strings`, where a list of endpoints/connection-strings can be provided. The endpoints/connection-strings will be attempted to connect in priority order.
```
Comment (zhenlan) [inline releaseNotes/SpringCloudAzureAppConfigurationConfig.md:22]: It's not necessary about the origin at all.
 
> Added support for Geo-Replication, enables the library to connect to a replica when the origin store fails to connect.

Added failover support for App Configuration stores with geo-replication enabled.

**Review by zhenlan — APPROVED** [review]

**Pull request merged**